scripts/covid19indiaORG/README.md

```diff
@@ -0,0 +1,104 @@
+# Importing covid19india.org Data Into Data Commons
```

Comment:
IIUC, this is only district level, even though the dataset includes state? If that's right, can you make that clear somewhere here? Thanks!

Reply:
This import includes both district and state data. I will make a note. Thanks!